src/test/resources/org/prebid/server/it/openrtb2/gamma/test-gamma-bid-response.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "test-request-id",
+  "seatbid": [
+    {
+      "seat": "958",
+      "bid": [
+        {
+          "id": "7706636740145184841",
+          "impid": "test-imp-id",
+          "price": 0.500000,
+          "adid": "29681110",
+          "adm": "some-test-ad",
+          "adomain": [
+            "sample.com"
+          ],
+          "cid": "958",
+          "crid": "29681110",
+          "h": 250,
+          "w": 300
+        }
+      ]
+    }
+  ],
+  "bidid": "5778926625248726496",
+  "cur": "USD"
+}
+}
```

Comment:
No harm, but please, remove an extra closing curly brace and an extra line - we don't need one in .json format.